templates/partials/_publisher_in_search.html

```diff
@@ -0,0 +1,10 @@
+<div class="card border-white" style="max-width: 150px;">
+    <a href="{% url 'publisher-detail-page' publisher.slug %}" class="text-decoration-none">
+        <img class="card-img-top card-img-150"
+            src="{% if publisher.logo %}{{ publisher.log.url }}{% else %}/static/images/person-outline.png{% endif %}"
```

Comment:
Do we need to add separated alternative image for publisher? I mean instead of `/static/images/person-outline.png`

Reply:
Ideally - yes. Practically - no. Most likely all publishers will have logos as we are going to add them gradually. With people it's harder to find photos of everyone.